# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/examples/common/boilerplate.rs
# Commit: 0415da2e9198
# Full Hash: 0415da2e9198c74c8881269e600dca28f38b8819
# Author: Glenn Watson <gw@intuitionlibrary.com>
# Date: 2020-04-14 04:09:24
# Regressor Bug: 1629693
# File Overlap Count: 1
# Description:
#   Bug 1629693 - Enable GPU cache scatter path when running ANGLE. r=kvark
#   
#   On some (mostly older, integrated) GPUs, the normal GPU texture cache
#   update path doesn't work well when running on ANGLE, causing CPU stalls
#   inside D3D and/or the GPU driver. To reduce the number of code paths we
# ==============================================================================

diff -r 17c0b08918ac -r 0415da2e9198 gfx/wr/examples/common/boilerplate.rs
--- a/gfx/wr/examples/common/boilerplate.rs	Mon Apr 13 21:38:51 2020 +0000
+++ b/gfx/wr/examples/common/boilerplate.rs	Mon Apr 13 23:26:07 2020 +0000
@@ -164,7 +164,6 @@
         precache_flags: E::PRECACHE_SHADER_FLAGS,
         device_pixel_ratio,
         clear_color: Some(ColorF::new(0.3, 0.0, 0.0, 1.0)),
-        //scatter_gpu_cache_updates: false,
         debug_flags,
         //allow_texture_swizzling: false,
         ..options.unwrap_or(webrender::RendererOptions::default())